### PR TITLE
[repository.lic] v2.47 show-updatable output update

### PR DESCRIPTION
**Pull request by mrhoribu**
update show-updatable output to list author & game for scripts

### CODE REVIEW

File: scripts/repository.lic [hunk 1]
```diff
@@ -1466,16 +1477,23 @@ elsif (cmd[0] =~ /^show-?updat(?:e|able)$/i)
...
+    Settings['updatable'][:scripts].sort_by { |script| script[:filename] }.each { |script_updatable|
+      table_rows.push([script_updatable[:filename], script_updatable[:author], script_updatable[:game]])
+    }
   end
-  output.concat "\n"
-  respond output
+  table = Terminal::Table.new(
+    rows: table_rows
+  )
+  _respond "<output class=\"mono\"/>\n" + table.to_s + "\n<output class=\"\"/>"
```
Comment (FarFigNewGut) [inline scripts/repository.lic:1496]:
```
table_str = table.to_s
table_str.gsub(/Alastir/, "#{monsterbold_start}Alastir#{monsterbold_end}")
_respond "<output class=\"mono\"/>\n" + table_str + "\n<output class=\"\"/>"
```

Comment (mrhoribu) [reply scripts/repository.lic:1496]: Uncertain if I agree with monsterbolding a name. Does it mean a good thing or a bad thing to someone unfamiliar to the reasoning of the monsterbold. Also not sure I want to take on the moderation, and never ending battle that comes with it, of an open source system that right now the server for it are not all controlled by the community but by a single server owner (Tillmen).

Comment (FarFigNewGut) [reply scripts/repository.lic:1496]: i was (half) kidding. More like 10%.

**Comment by OSXLich-Doug** [comment]
Good change, reviewed - this is ready.

**Pull request merged**